libraries/ESP8266WebServer/src/ESP8266WebServer-impl.h

```diff
@@ -618,6 +619,20 @@ void ESP8266WebServerTemplate<ServerType>::collectHeaders(const char* headerKeys
   }
 }
 
+template<typename ServerType>
+void ESP8266WebServerETagTemplate<ServerType>::collectHeaders(const char* headerKeys[], const size_t headerKeysCount) {
+  WST::_headerKeysCount = headerKeysCount + 2;
```

Comment:
Isn't the `WST::` bit implied for the class local variables?  Others template methods seem to be fine w/o it...

Reply:
`ESP8266WebServerETagTemplate<ServerType>` inherit form `ESP8266WebServerTemplate<ServerType>`, the variables that I'm trying to access are from the parent class: `WTS` is a [shorthand](https://github.com/esp8266/Arduino/pull/7709/files/ae3d925e648743486cf07b7d84b8355e72c6b111#diff-799c46c81de7a8939be0ec4f4aef5ad592378592d337950447996243b78f37e0R307) for `ESP8266WebServerTemplate<ServerType>`

I went for the full baseclass templated name, I could also use `this->` to access the parent's variables but [they are not plainly accessible in the current scope](https://stackoverflow.com/questions/31231875/cant-access-variable-in-template-base-class)

Is `this->` more readable?

Reply:
Thanks for clearing this up!